src/compose.js

```diff
@@ -10,11 +10,16 @@
  */
 
 export default function compose(...funcs) {
-  if (funcs.length === 0) {
+  const { length } = funcs
+  if (!length) {
```

Comment:
I'd remove this variable extraction. You don't use it below. It's an unnecessary optimization.